tests/spec/core/webidl-spec.js

```diff
@@ -1218,4 +1218,79 @@ callback CallBack = Z? (X x, optional Y y, /*trivia*/ optional Z z);
     const banana = p.querySelector("dfn");
     expect(banana.dataset.export).not.toBeDefined();
   });
+  it("autolinks partial definitions", async () => {
+    const body = `
+      <section data-dfn-for="EventInit">
+        <p>
+          <dfn>Banana</dfn>
+          <dfn>itWorks</dfn>
+        </p>
+        <pre class="idl">
+          // Local ref
+          interface Banana {};
+          // Local ref
+          partial interface Banana {};
+          // DOM spec
+          partial interface mixin DocumentOrShadowRoot {};
+          // Fetch spec
+          partial interface Request {};
+          // DOM spec
+          partial dictionary EventInit {
+            boolean itWorks;
+          };
+        </pre>
+      </section>
+    `;
+    const ops = makeStandardOps({ xref: "web-platform" }, body);
+    const doc = await makeRSDoc(ops);
+    const [
+      banana,
+      bananaPartial,
+      docOrShadowMixin,
+      requestPartialInterface,
+      eventInitDict,
+      ,
```

Comment:
```suggestion
```

A mistake?

Reply:
nope, that one skips "boolean"

Reply:
Can you suggest a comment // skip "boolean" link?

Reply:
Ah, oops, right, sorry!